doc/rados/configuration/osd-config-ref.rst

```diff
@@ -654,7 +654,7 @@ perform well in a degraded state.
               increased load on the cluster.
 
 :Type: 32-bit Integer
-:Default: ``15``
+:Default: ``3``
```

Comment:
could you add a "`Signed-off-by`" line at the end of your commit message? "git commit -s" will do the trick for you. see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#1-sign-your-work, and your `Fixes` line does not look right, because this number will clash with the PR number in future. so you commit message might need to be revised so it looks like
```
doc: Change the default values of some OSD options

These values were changed to reflect this change a3f9cba

Fixes: http://tracker.ceph.com/issues/20199
Signed-off-by: Bara Ancincova <bara@redhat.com>
```